lib/realtime/tenants/repo/migrations/20240919140541_reduce_grants_postgres_user.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Realtime.Tenants.Migrations.ReduceGrantsPostgresUser do
+  @moduledoc false
+  use Ecto.Migration
+
+  def change do
+    execute("revoke supabase_realtime_admin from postgres")
+    execute("alter default privileges for role supabase_admin in schema realtime revoke all on tables from postgres")
+    execute("alter default privileges for role supabase_admin in schema realtime revoke all on functions from postgres")
+    execute("alter default privileges for role supabase_admin in schema realtime revoke all on sequences from postgres")
```

Comment:
this actually breaks our tests and implementations as `supabase_admin` is the user we use to connect to tenant database and we still need them 😭 

so I will need to remove this 3 `execute` @soedirgo

Reply:
Hmm how does it break? It shouldn't affect existing tables/functions

Reply:
running migrations fails:
```
     18:14:41.532 project=dev_tenant external_id=dev_tenant [error] MigrationsFailedToRun: %Postgrex.Error{
       message: nil,
       postgres: %{
         code: :undefined_object,
         line: "5101",
         message: "role \"supabase_admin\" does not exist",
         file: "acl.c",
         unknown: "ERROR",
         severity: "ERROR",
         pg_code: "42704",
         routine: "get_role_oid"
       },
       connection_id: 156,
       query: nil
     }
```

Reply:
Oh we granted the default privileges [here](https://github.com/supabase/postgres/blob/092d9702ac7cffd8938f24fcbcd131b1aa470333/migrations/db/migrations/20211130151719_update-realtime-permissions.sql#L4) 😭 then yeah we need to move these out of Realtime tenant migrations